sdk/lib/gcc-compat/StringCchCopyNA.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * PROJECT:     GCC c++ support library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     StringCchCopyNA implementation
+ * COPYRIGHT:   Copyright 202024 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+#include <windows.h>
+#define StringCchCopyNA _StringCchCopyNA
+#include <strsafe.h>
+
+#undef StringCchCopyNA
+
+HRESULT
+WINAPI
+StringCchCopyNA(
```

Comment:
Which code in there requires this function? It doesn't appear to be used, at least explicitly.

Reply:
GCC's c++ library.

Reply:
strange that they use it (I would have expected they use the posix string funcs instead :rofl: )